content/docs/user-guide/experiment-management/persisting-experiments.md

```diff
@@ -0,0 +1,99 @@
+# Persisting Experiments
+
+When you run an experiment with `dvc exp run`, it produces temporary objects in
+the Git repository. These are listed with `dvc exp list` and `dvc exp show`.
```

Comment:
The part about temporary objects is an implementation detail that we should put in https://dvc.org/doc/user-guide/experiment-management/running-experiments probably. I don't think it's relevant info here.

Reply:
Removed that too.